org.flightgear.FlightGear.yaml

```diff
@@ -108,13 +109,13 @@ modules:
       - -DCMAKE_BUILD_TYPE=Release
     sources:
       - type: archive
-        url: https://github.com/openscenegraph/OpenSceneGraph/archive/OpenSceneGraph-3.6.5.tar.gz
-        sha256: aea196550f02974d6d09291c5d83b51ca6a03b3767e234a8c0e21322927d1e12
+        url: https://gitlab.com/flightgear/openscenegraph/-/archive/OpenSceneGraph-3.6.5/openscenegraph-OpenSceneGraph-3.6.5.tar.gz
+        sha256: c2c71bfaaf9ed28e28d763f94e361f7cfa2b026dc168f45f24ff391f0de9314e
```

Comment:
don't see the point to change this? https://gitlab.com/flightgear/openscenegraph/ is a mirror of https://github.com/openscenegraph/OpenSceneGraph/

and how did the hash change for the same version?

Reply:
I have no idea too why the hash of the FG version is different.

The motivation behind my decision is based on this text: "Because the OSG project is no longer officially maintained, current versions of FlightGear require a forked version of OSG that comes with several improvements and fixes. The fork is hosted on https://gitlab.com/flightgear/openscenegraph, with a specific branch for specific versions of FlightGear, as outlined in the table. "

Source: https://wiki.flightgear.org/OpenSceneGraph

However I'm abstaining from using a branch version without a tag. The table below this text suggests which is the correct branch for FG 2024.x.x versions but it doesn't have a tag or an official release and OSG 3.6.5 works fine with it.

Reply:
Looks like due to a change in casing of the folder inside the archive.